### PR TITLE
8266749: AArch64: Backtracing broken on PAC enabled systems

### PR DESCRIPTION
**Pull request by a74nh**
Backport-of: ae986bc8dff92a77e91e6ee640aa27c68abb8def

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8266749](https://bugs.openjdk.java.net/browse/JDK-8266749): AArch64: Backtracing broken on PAC enabled systems

### Reviewers
 * [Paul Hohensee](https://openjdk.java.net/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk11u-dev pull/39/head:pull/39` \
`$ git checkout pull/39`

Update a local copy of the PR: \
`$ git checkout pull/39` \
`$ git pull https://git.openjdk.java.net/jdk11u-dev pull/39/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 39`

View PR using the GUI difftool: \
`$ git pr show -t 39`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk11u-dev/pull/39.diff">https://git.openjdk.java.net/jdk11u-dev/pull/39.diff</a>

</details>

### CODE REVIEW

**Comment by a74nh** [comment]
Identical to original commit except that the new windows and macos files have been removed.

**Comment by a74nh** [comment]
> Issue is not open.
Do I need to create a new issue for the backport? The instructions at https://wiki.openjdk.java.net/display/SKARA/Backports are not clear.

**Comment by jerboaa** [comment]
> > Issue is not open.
> 
> Do I need to create a new issue for the backport? The instructions at https://wiki.openjdk.java.net/display/SKARA/Backports are not clear.

Changing the PR title to: `Backport ae986bc8dff92a77e91e6ee640aa27c68abb8def` should fix it.

**Comment by jerboaa** [comment]
Not the commit message. The PR title here. Yes, confusing, I know.

**Comment by a74nh** [comment]
> Not the commit message. The PR title here. Yes, confusing, I know.

Thanks :) think I got it right this time.

**Review by phohensee — APPROVED** [review]
Lgtm.

**Comment by RealCLanger** [comment]
@a74nh you still need to wait for backport approval in JBS before integrating... But, nevermind, I've just given it.

**Pull request closed without merging**
